### DISCUSSION TITLE
Como adicionar seleção de linhas em uma Page Dynamic Table?

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1727] [Tire suas dúvidas] [answered]
Olá, como posso fazer para adicionar uma coluna para seleção de linhas em um componente Page Dynamic Table? 

Similar ao exemplo a seguir:

![image](https://github.com/po-ui/po-angular/assets/46979335/daa63a63-465b-4a59-9b02-dc9328176532)

A única forma que consegui habilitar é através da opção removeAll do PoPageDynamicTableActions, mas não quero que a ação seja para excluir linhas.

Agradeço desde já pela ajuda.

**Comment by jhosefmarks** [comment, marked as answer]
Você pode criar uma ação customizada (PoPageDynamicTableCustomAction) com a opção selectable ativa.

Veja o exemplo [PO Page Dynamic Table - Hotels](https://po-ui.io/documentation/po-page-dynamic-table?view=web)
